app/docker/views/services/edit/serviceController.js

```diff
@@ -402,6 +402,7 @@ function ($q, $scope, $transition$, $state, $location, $timeout, $anchorScroll,
     service.RestartDelay = ServiceHelper.translateNanosToHumanDuration(service.RestartDelay) || '5s';
     service.RestartWindow = ServiceHelper.translateNanosToHumanDuration(service.RestartWindow) || '0s';
     service.UpdateDelay = ServiceHelper.translateNanosToHumanDuration(service.UpdateDelay) || '0s';
+    service.StopGracePeriod = (service.StopGracePeriod ? ServiceHelper.translateNanosToHumanDuration(service.StopGracePeriod) : '');
```

Comment:
I believe that the `(` & `)` chars are optional here.

E.g.: `service.StopGracePeriod = service.StopGracePeriod ? ServiceHelper.translateNanosToHumanDuration(service.StopGracePeriod) : '';`